src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -19,6 +19,7 @@ export interface MessageData {
   placement: "left" | "right" | "center";
   timestamp?: Date;
   content: React.ReactNode;
+  index?: number;
```

Comment:
Just for my understanding, the intent is that we'd pass in the `index` on the Twilio message object? I have't read the documentation, but I see Twilio's message object does have an `index` field so I'd assume that meets our needs.

Reply:
Yes thats exactly right! Done [here](https://github.com/Clever/launchpad/pull/3503/commits/1b3133a7de713ae02883054edcf3f51878aa5352#diff-5f76153f13f1f361d2dff59c57a95a7aR32)